Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/README.md

```diff
@@ -0,0 +1,1540 @@
+>| 12898 | Botnet malware | These emails impersonate BBVA Compass, claiming to deliver payment information and make the request to view the attached file. The attachment is a Microsoft Office Word document containing a malicious Office macro tasked with downloading and executing a sample of the Ursnif malware. Ursnif is a data stealer, exfiltrating keystroke logs, browsing history, and other potentially sensitive information via its command and control location. | Paid/Closed Heloc Letter - OfficeMacro, Ursnif | 2018-08-28 14:46:23 |
+>| 13962 | DDoS Capable multi-functional RAT | These messages claim to contain a quotation request in the attached file. The file is a .zip archive containing a Windows shortcut .lnk file. Interacting with the shortcut downloads and executes a PowerShell script. When run, the PowerShell script downloads and displays a decoy image before decoding and running an embedded executable. The executable is a sample of the DarkComet Remote Access Trojan. This malware is tasked with providing the threat actors with remote access to the victim's machine, as well as the capability to exfiltrate sensitive data via its command and control location. Additional .lnk payload locations were found during the course of this analysis. | Quotation Request - LNK Downloader, Malicious Powershell Script, Dark Comet RAT | 2018-10-12 12:28:46 |
+>| 14078 | Malware capable of collecting victim keystrokes for exfiltration | These messages claim to have information about a shipment from DHL available to recipients via a link embedded i the message body. Clicking the link downloads a Microsoft Office document containing a hostile macro script used to decode and run an embedded executable sample of the Limitless Keylogger. This keylogger has a number of capabilities that make it a robust espionage tool. In addition to being able to dump stored and session credentials, this malware is capable of taking screenshots of user activity and collecting keystrokes from the Windows environment. It is also capable of spreading via many different methods including removable USB devices and instant messaging. This malware exfiltrates this information to the threat actors' command and control location via email. | Package Notification - OfficeMacro, Limitless Keylogger | 2018-10-16 17:46:45 |
+>| 14794 | Adaptable financial crimes botnet trojan with email worm and malware delivery capabilities, also known as Emotet<br/>Advanced banking trojan utilizing both webinjects and browser redirection | Multilingual invoice themed messages deliver Microsoft Word documents containing Office macros which download Geodo malware. Geodo then downloads and runs the banking trojan IcedID. | Invoices - OfficeMacro, Geodo, IcedID | 2018-11-08 20:13:04 |
+>| 14943 | Information stealer designed to steal stored credentials and cryptocurrency wallets | Receipt themed emails deliver AZORult malware. | CTM Receipt - AZORult | 2018-11-13 15:55:51 |
```

Comment:
lets trim is down to:
1. only 2 lines.
2. cut the text - you can make up your text here.